rosdep/sources.list.d/20-default.list

```diff
@@ -1,10 +1 @@
-# os-specific listings first
-yaml https://raw.githubusercontent.com/ros/rosdistro/master/rosdep/osx-homebrew.yaml osx
-
-# generic
-yaml https://raw.githubusercontent.com/ros/rosdistro/master/rosdep/base.yaml
-yaml https://raw.githubusercontent.com/ros/rosdistro/master/rosdep/python.yaml
-yaml https://raw.githubusercontent.com/ros/rosdistro/master/rosdep/ruby.yaml
-gbpdistro https://raw.githubusercontent.com/ros/rosdistro/master/releases/fuerte.yaml fuerte
```

Comment:
do we not need the `yaml ...` in front of the filename?

Reply:
We do. Thanks for catching this! It doesn't work yet till merging https://github.com/RoboStack/ros-noetic/pull/119 by the way.